jerry-core/ecma/builtin-objects/ecma-builtin-string.c

```diff
@@ -291,19 +291,20 @@ ecma_builtin_string_object_from_code_point (ecma_value_t this_arg, /**< 'this' a
 
   for (ecma_length_t index = 0; index < args_number; index++)
   {
-    ecma_value_t to_number_value = ecma_op_to_number (args[index]);
+    ecma_number_t to_number_num;
+    ecma_value_t to_number_value = ecma_op_to_number (args[index], &to_number_num);
```

Comment:
This could be removed, the `ecma_op_to_integer` call below will call to_number anyways.

Reply:
This request is still valid.